cmd/fuzztool/fuzztool.go

```diff
@@ -187,21 +186,20 @@ func copyCorpus(srcRoot, dstRoot string) {
 	}
 }
 
-func main() {
-	debugF := flag.Bool("debug", false, "enable debug mode")
-	srcF := flag.String("src", "", "source, one of: 'seed', 'generated', or collected corpus' directory")
-	dstF := flag.String("dst", "", "destination, one of: 'seed', 'generated', or collected corpus' directory")
-	flag.Parse()
+var CLI struct {
+	Corpus struct {
+		Src string `arg:"" help:"source, one of: 'seed', 'generated', or collected corpus' directory"`
+		Dst string `arg:"" name:"dst" help:"destination, one of: 'seed', 'generated', or collected corpus' directory"`
+	} `cmd:""`
+	Debug bool `help:"enable debug mode"`
+}
 
-	if flag.NArg() != 0 {
-		flag.Usage()
-		fmt.Fprintln(flag.CommandLine.Output(), "zero arguments expected")
-		os.Exit(2)
-	}
+func main() {
+	ctx := kong.Parse(&CLI)
 
 	logging.Setup(zap.InfoLevel)
 
-	if *debugF {
+	if CLI.Debug {
```

Comment:
Do you have an example of how Debug could be set? Is it something like `go run {{.RACEFLAG}} ./cmd/fuzztool/fuzztool.go debug corpus generated {{.FUZZCORPUS}}`? Or is it a flag? Should debug be mentioned in a particular place in such a command line?

Reply:
It's a flag
```
Usage: fuzztool <command>

Flags:
  -h, --help     Show context-sensitive help.
      --debug    enable debug mode

Commands:
  corpus <src> <dst>

Run "fuzztool <command> --help" for more information on a command.
```

Reply:
May we make `--help` and `--debug` help texts look similar to each other? One of them is a full sentence (starts with a capital letter, ends with a dot), but not the other

Reply:
@noisersup the dot at the end of the sentence is still missing!

Reply:
My bad
```
go run ./cmd/fuzztool/fuzztool.go -h
Usage: fuzztool <command>

Flags:
  -h, --help     Show context-sensitive help.
      --debug    Enable debug mode.

Commands:
  corpus <src> <dst>

Run "fuzztool <command> --help" for more information on a command.
```
```
go run ./cmd/fuzztool/fuzztool.go corpus -h
Usage: fuzztool corpus <src> <dst>

Arguments:
  <src>    Source, one of: 'seed', 'generated', or collected corpus' directory.
  <dst>    Destination, one of: 'seed', 'generated', or collected corpus' directory.

Flags:
  -h, --help     Show context-sensitive help.
      --debug    Enable debug mode.
```